admin_guide/high_availability.adoc

```diff
@@ -153,29 +153,52 @@ $ ping hello-openshift.shard1.v3.rhcloud.com
 [[configuring-ip-failover]]
 == Configuring IP Failover
 
-Using IP failover involves switching IP addresses to a redundant or stand-by
-set of nodes on failure conditions.
+IP Failover is configured using the `oadm ipfailover` command with suitable options.
+`oadm ipfailover` creates a DC (Deployment Controller) based on provided options.
```

Comment:
Maybe `An IP Failover configuration is created using the oadm ipfailover command.` and add a link to the options and then maybe say `Internally this command creates a deployment config using the provided options`?